<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmGameWindow</class>
 <widget class="QWidget" name="FrmGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 127, 255), stop:1 rgba(0, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font-size: 30px;
font-weight: 400;
color: white;</string>
   </property>
   <property name="text">
    <string>Picadvisor Logo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSolution">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter your solution...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSolution">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font-size: 25px;
font-weight: 400;
color: white;</string>
   </property>
   <property name="text">
    <string>What shows the picture?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTimer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
color: 'white';
font-size: 25px;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSolution_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
font-size: 25px;
font-weight: 400;
color: white;</string>
   </property>
   <property name="text">
    <string>Time is running:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblShowImage">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>525</width>
     <height>525</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
